data/reference/syntax.adoc

```diff
@@ -0,0 +1,267 @@
+
+. Step 1
+. Step 2
+.. Step 2a
+.. Step 2b
+. Step 3
+
+=== Checklist
+
+* [*] checked
+* [x] also checked
+* [ ] not checked
+*     normal list item
+
+=== Description
+
+first term:: definition of first term
+second term:: definition of second term
+
+== Document Structure
+
+=== Document Header
+
+= My Document's Title
+Doc Writer <doc.writer@domain.org>
+v1.0, 2019-01-01
+
+My document provides...
+
+=== Section titles
+
+= Document Title (Level 0)
+== Level 1 Section Title
+=== Level 2 Section Title
+==== Level 3 Section Title
+===== Level 4 Section Title
+====== Level 5 Section Title
+== Another Level 1 Section Title
+
+=== Include files
+
+include::basics.adoc[]
+include::https://domain.org/installation.adoc[]
+
+== Delimited Blocks
+
+////
+Comment
+////
+
+[NOTE]
+// one of: NOTE, TIP, WARNING, IMPORTANT, CAUTION
+====
+admonition
+====
+
+[example]
+====
+example
+====
+
+```language
+fenced
+```
+
+[listing]
+----
+listing
+----
+
+[literal]
+....
+literal
+....
+
+--
+open
+--
+
+[pass]
+++++
+passthrough
+++++
+
+[quote, attribution, citation title and information]
+____
+quote
+____
+
+[sidebar]
+****
+sidebar
+****
+
+[source, language, options]
+----
+source
+----
+
+[stem]
+++++
+stem
+++++
+
+|===
+table
+|===
+
+[verse]
+____
+verse
+____
+
+== Tables
+
+.An example table
+[width="40%", cols="^,2m", frame="topbot", options="header,footer"]
+|===
+|Column 1 |Column 2
+|1        |Item 1
+|2        |Item 2
+|3        |Item 3
+|6        |Three items
+|===
+
+Common attributes:
+
+grid:    none, cols, rows, all
+frame:   topbot, none, sides, all
+options: header, footer
+format:  psv, csv, dsv
+valign:  top, bottom, middle
+width:   1%..100%
+cols:    colspec[,colspec,...]
+
+colspec:    [multiplier*][align][width][style]
+multiplier: 1...
+width:      1... or 1%...100%
+align:      [horiz][.vert]
+           horiz: < (left), ^ (center), > (right)
+           vert:  < (top),  ^ (middle), > (bottom)
+style:      d[efault], e[mphasis], m[onospaced], a[sciidoc],
+          s[trong], l[iteral], v[erse], h[eader]
+cell:       [cellspec]|data
+cellspec:   [span*|+][align][style]
+span:       [colspan][.rowspan]
+           colspan: 1...
+           rowspan: 1...
+
+== Admonition
+
+NOTE: Please note that...
+
+TIP: Pro tip...
+
+IMPORTANT: Don't forget...
+
+WARNING: Watch out for...
+
+CAUTION: Ensure that...
+
+== Images
+
+image::path/to/file.jpg[caption, width, height]
+
+inline image:path/to/file.png[caption, title="title"].
+
+== Videos
+
+video::path/to/file.mp4[width=640, start=60, end=140, options=autoplay]
+
+video::rPQoq7ThGAU[youtube]
+
+video::67480300[vimeo]
+
+== Source Code
+
+=== Code block with title, syntax highlighting and callouts
+
+.app.rb
+[source, ruby]
+----
+require 'sinatra' // <1>
+
+get '/hi' do // <2>
+  "Hello World!" // <3>
+end
+----
+<1> Library import
+<2> URL mapping
+<3> HTTP response body
+
+=== Code block sourced from file
+
+[source, ruby]
+----
+include::app.rb[]
+----
+
+== Horizontal Rules
```

Comment:
These are now thematic breaks, not horizontal rules.

We might even want to group these under "Breaks" using a comment line above like this:

```
== Breaks

// thematic break (aka horizontal rule)
---

// page break
<<<
```